life_expectancy/region.py

```diff
@@ -0,0 +1,36 @@
+from enum import Enum, auto
+
+
+class RegionType(Enum):
+    COUNTRY = auto()
+    REGION = auto()
+
+
+class Region(Enum):
+    PT = (RegionType.COUNTRY, "PT")
+    ES = (RegionType.COUNTRY, "ES")
+    EU28 = (RegionType.REGION, "EU28")
+    EFTA = (RegionType.REGION, "EFTA")
```

Comment:
Lazy lazy! Didn't include all regions!

(Next time I'm going to create an assignment for people add all regions using VSCode's multi-select so they can make better use of the Text Editors module!)